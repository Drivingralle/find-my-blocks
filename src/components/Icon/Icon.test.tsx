import React from "react";
import renderer from "react-test-renderer";
import { Icon } from "./";

describe("Icon component should", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Icon icon="camera" />).toJSON;
    expect(tree).toMatchSnapshot();
  });

  it("renders custom sizes", () => {
    const tree = renderer.create(<Icon icon="camera" size={100} />).toJSON;
    expect(tree).toMatchSnapshot();
  });

  it("renders custom colors", () => {
    const tree = renderer.create(<Icon icon="camera" color="#ff0000" />).toJSON;
    expect(tree).toMatchSnapshot();
  });
});
