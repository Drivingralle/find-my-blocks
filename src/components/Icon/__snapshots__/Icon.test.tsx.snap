// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon component should renders correctly 1`] = `[Function]`;

exports[`Icon component should renders custom colors 1`] = `[Function]`;

exports[`Icon component should renders custom sizes 1`] = `[Function]`;
